marketplace/src/main/assemble/assembly.xml

```diff
@@ -53,26 +54,68 @@
           <artifact scope="provided" />
           <artifact type="pom" />
         </excludes>
-      </artifact:set>
+      </artifact:dependencies>
     </copy>
     <!-- Third-party libraries -->
     <copy todir="${outdir}/nxr/lib" overwrite="true">
-      <artifact:set>
+      <artifact:dependencies key="org.nuxeo.ecm.platform:nuxeo-agenda-core:::">
         <includes>
           <artifact groupId="!org.nuxeo*" />
           <artifact groupId="org.nuxeo.lib*" />
         </includes>
         <excludes>
           <artifact scope="test" />
         </excludes>
-      </artifact:set>
+      </artifact:dependencies>
     </copy>
     <nx:rmdups dir="${outdir}/nxr/lib" />
+    <!-- TODO What should be unzipped / zipped before `nx:zipdiff`? Not sure what I did is correct. -->
```

Comment:
Associating a _TODO_ with a JIRA ticket helps to identify which issue this task belongs to. This is a good practice. :+1:

Thanks for this hint and the explanation!